core/src/main/kotlin/org/evomaster/core/problem/rest/service/RestSampler.kt

```diff
@@ -594,4 +772,80 @@ class RestSampler : Sampler<RestIndividual>() {
                 }
     }
 
+
+
+    fun lcs(a: String, b: String): String {
```

Comment:
what does lcs stand for? best to use full names. Remember that, in IntelliJ, there is auto-completion... so long descriptive names are not a problem :)